labelme/widgets/canvas.py

```diff
@@ -315,49 +319,52 @@ def mousePressEvent(self, ev):
                         self.drawingPolygon.emit(True)
                         self.update()
             else:
-                self.selectShapePoint(pos)
+                group_mode = (int(ev.modifiers()) == QtCore.Qt.ControlModifier)
+                self.selectShapePoint(pos, multiple_selection_mode=group_mode)
                 self.prevPoint = pos
                 self.repaint()
         elif ev.button() == QtCore.Qt.RightButton and self.editing():
-            self.selectShapePoint(pos)
+            group_mode = (int(ev.modifiers()) == QtCore.Qt.ControlModifier)
+            self.selectShapePoint(pos, multiple_selection_mode=group_mode)
             self.prevPoint = pos
             self.repaint()
 
     def mouseReleaseEvent(self, ev):
         if ev.button() == QtCore.Qt.RightButton:
-            menu = self.menus[bool(self.selectedShapeCopy)]
+            menu = self.menus[min(len(self.selectedShapesCopy), 2)]
```

Comment:
When I tried this. `len(self.selectedShapesCopy)` is always `0` because selection is reset in the `mouseSelectionEvent`.
How did you test this?

Reply:
SelectedShapeCopy is filled when you start to grag mouse, so you get "Move here" and "Copy here" only if you drag objects an drop them. Befeore it menu[0] seems to be normal.

Reply:
Got it. Thanks.